source/slang/slang-check-decl.cpp

```diff
@@ -7076,6 +7076,13 @@ namespace Slang
             {
                 args.add(DeclRefType::create(astBuilder, astBuilder->getDirectDeclRef(genericTypeParamDecl)));
             }
+            else if (auto genericTypePackParamDecl = as<GenericTypePackParamDecl>(mm))
+            {
+                auto packType = DeclRefType::create(astBuilder, astBuilder->getDirectDeclRef(genericTypePackParamDecl));
+                //auto eachType = astBuilder->getEachType(packType);
+                //auto expandType = astBuilder->getExpandType(eachType, makeArrayViewSingle(packType));
```

Comment:
I don't quite understand what the comments are supposed to refer to?

Reply:
Sorry, this is left over code that should be deleted.